src/MonoTorrent.Client/MonoTorrent/TorrentCreator.cs

```diff
@@ -516,6 +537,25 @@ static BEncodedValue ToFileInfoDict (InputFile file)
             return fileDict;
         }
 
+        static BEncodedValue ToPaddingFileInfoDict (InputFile file)
+        {
+            var fileDict = new BEncodedDictionary ();
+
+            var filePath = new BEncodedList ();
+            filePath.Add (new BEncodedString (".pad"));
+            filePath.Add (new BEncodedString ($"{file.Padding}"));
+
+            fileDict["length"] = new BEncodedNumber (file.Padding);
+            fileDict["path"] = filePath;
+
+            // TODO JMIK: do padding files have MD5 hashes? It's easy to add an all-zero MD5 hash calculation here
+            //if (file.MD5 != null)
+            //    fileDict["md5sum"] = (BEncodedString) file.MD5;
```

Comment:
A library which wants to validate the md5 *and* doesn't support bep47/bep52 would need a valid MD5. Hashing all zeros seems fine?